xml/depl_conf_admin_crowbar.xml

```diff
@@ -1439,17 +1439,16 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
    <example xml:id="ex.conduits.machine">
     <title>Network Modes for Certain Machines</title>
     <para>
-     Apart from the roles listed under <xref linkend="ex.conduits.role"/>
+     Apart from the roles listed under <xref linkend="ex.conduits.role"/>,
      each node in &cloud; has a unique role, which lets you create modes
      matching exactly one node. The role is named after the scheme
-     <literal>crowbar-d <replaceable>FULLY QUALIFIED
+     <literal>crowbar-<replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable></literal>. The <replaceable>FULLY QUALIFIED
-     HOSTNAME</replaceable> in turn is composed of the following: the MAC address of the
-     network interface used to boot the node via PXE, and the domain name
-     configured on the &admserv;. Colons and periods are replaced with
-     underscores. An example role name would be:
-     <literal>crowbar-d1a-12-05-1e-35-49_my_cloud</literal>.
+     HOSTNAME</replaceable> in turn is composed of the following: a prefix 'd', the MAC address of the network interface used to boot the node via PXE, and the domain name configured on the &admserv;. Colons and periods are replaced with
+     underscores. An example role name would be: 
```

Comment:
This needs to be changed - I already had it wrong in the original text. It mixes how the FQDN and the role name are created, and contains misleading information.

* the FQDN is created by prefixing "d" to the MAC address and replacing colons with dashes
* the role name is created by prefixing the FQDN with "crowbar-" and replacing the periods in the domain name with underscores (the host name part is _not_ changed)